app/uk/gov/hmrc/helptosavefrontend/config/FrontendAppConfig.scala

```diff
@@ -95,4 +95,5 @@ class FrontendAppConfig @Inject() (override val runModeConfiguration: Configurat
   val reportAProblemNonJSUrl: String = s"$contactBaseUrl/contact/problem_reports_nonjs?service=$contactFormServiceIdentifier"
   val betaFeedbackUrlNoAuth: String = s"$contactBaseUrl/contact/beta-feedback-unauthenticated?service=$contactFormServiceIdentifier"
   val govUkExitURL: String = getString("gov-uk.url")
+  val govUkNotEligibleURL: String = getString("gov-uk.not-eligible.url")
```

Comment:
We could add something like ```val notEligibleUrl: String = s"${getUrlFor("help-to-save-frontend")}/not-eligible" ```

Reply:
removed the hard coded full stop.

Reply:
link text updated to correct value as per story www.gov.uk/get-help-savings-low-income/eligibility